src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,33 @@
 
     @Override
     public Actor add(Actor actor) {
```

Comment:
You don't need to set an ID for your entity explicitly. Hibernate will do it for you. Ensure that the Actor entity is correctly annotated and that the ID generation strategy is set up.